src/client/components/Meal/Meal.jsx

```diff
@@ -1,8 +1,53 @@
-import React from "react";
-export default function Meal({ title, description }) {
+import React, { useContext, useState } from "react";
+import mealStyles from "./Meal.module.css";
+import { MealsContext } from "../../providers/MealsContextProvider";
+
+export default function Meal({ mealId }) {
+  const [meals, dispatchMeals] = useContext(MealsContext);
+  const meal = meals.find((meal) => meal.id === mealId);
+  const { title, description, location, when, price, created_date } = meal;
+  const [expanded, setExpanded] = useState(false);
+  const handleExpand = (e) => {
+    setExpanded(!expanded);
+  };
+  const mealDetailsClassnames = `${mealStyles.mealDetails} ${
+    expanded ? mealStyles.mealDetailsExpanded : ""
+  }`;
+  const mealWrapperStyles = `${mealStyles.meal} ${
+    expanded ? mealStyles.mealWrapperExpanded : ""
+  }`;
+
   return (
-    <p>
-      <b>{title}:</b> {description}
-    </p>
+    <div className={mealWrapperStyles}>
+      <div className={mealDetailsClassnames}>
+        <h2 className={mealStyles.title}>{title ?? "Meal title is empty"}:</h2>
+        <p>
+          <b>Description:</b> {description ?? "Meal description is empty"}
```

Comment:
Not that what you did is incorrect, but you might want to choose the [OR operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_OR#description), ||, over the [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing), ??.

Reply:
I just used newer feature... :) probably should have gone with ||. Could you explain, what downsides do you see in using ?? here? Maybe I don't understand some potential bugs, which could result from using  nullish coalescing operator? Cause I only needed to check for undefined. If they put an empty string inside some field, then probably it should be displayed as is... Or maybe not... :)